CONTRIBUTING.md

```diff
@@ -40,3 +49,105 @@ labels
+
+### Reviewer
+
+Once a community participant has demonstrated expertise in one GUAC topic area,
+owners and/on maintainers can propose graduating the community participant to
+the next role, that of Reviewer.
+
+A reviewer will be responsible of reviewing contributions from other
+contributors that touch the corresponding topic area. They should ensure that
+the PRs are adequately tested, follow the guidelines for the specific topic
+area.
+
+Since reviewer gain write access to the repository, they can also contribute
+directly, without needing to create a fork.
+
+### Owner
+
+A reviewer with significant review history and proven record of authorship in a
+specific GUAC topic area can be promoted to Owner status, upon a maintainer
+majority vote.
+
+The Owner is in full control over the specific topic area. They are expected to
+set direction and priorities, resolve technical trade-offs, prioritize between
+adding new features and handling technical debt. As such, they must show a deep
+understanding of technical problems involved in this area.
+
+Owners are invited to the GUAC owners and maintainers Slack channel.
+
+Owners will be added to `CODEOWNERS` file, so they will be notified of PRs that
+touch their area of interest.
+
+### Maintainer
+
+An owner that gains a deep understanding of GUAC architecture and design and can
+prove this by being an OWNER in at least 3 topic areas will be promoted to
+maintainer, after a n-1 approval vote from the other maintainers, according to
+the processes specified in [the governance document](./GOVERNANCE.md).
+
+A maintainer has almost full access to the repository and is invited to all
+maintainer meetings, where they can contribute input that will set direction and
+priorities of the entire project. They must have ability to commit to
+participating to at least the weekly maintainer sync meetings (currently 2
+meetings totalling up to 2 hours every week).
+
+A maintainer might be required to work with GUAC clients, thus they might be
+required to sign NDAs.
+
+The [GUAC governance document](./GOVERNANCE.md) also details the role of
+technical advisory member, which is offered to members of the community that
+have provided valuable input. The technical advisory members have a consulting
+role and are the only role without an activity requirement.
+
+### Inactivity, Stepping down
+
+Each role in the contribution ladder requires maintaining active contributions
+at the corresponding level. Inactivity is harmful to the project, so maintainers
+can decide to remove privileges from inactive contributors, after a majority
+vote. Inactivity is defined as periods longer than 3 months during which no
+contribution is being done.
+
+Contributors also have the option of stepping down voluntarily, by contacting
+the maintainers when circumstances affect future contribution potential. For
+highly active contributors that completely step away from the project, we will
+create an emeritus process.
```

Comment:
+1 , great addition here on emeritus process, want to keep the knowledge around.